Makefile

```diff
@@ -90,19 +90,13 @@ package: package-darwin-universal package-darwin-x86_64 package-darwin-arm64
 .PHONY: bump-version
 bump-version:
```

Comment:
You might consider moving to [rtx](https://github.com/jdx/rtx), like we've done in Tuist. The tool manages versions of dependencies deterministically and also provides a more feature-rich API for defining tasks and invoking them from the CLI ([example](https://github.com/tuist/tuist/blob/main/.rtx.toml#L9))